CMakeLists.txt

```diff
@@ -97,6 +99,23 @@ ExternalProject_Add(
     BUILD_BYPRODUCTS ${AOSCORE_BUILD_DIR}/lib/libaossmcpp.a ${AOSCORE_BUILD_DIR}/lib/libaosiamcpp.a
 )
 
+set(XEN_SRCDIR ${CMAKE_CURRENT_SOURCE_DIR}/../xen)
+set(XEN_BINDIR ${CMAKE_CURRENT_BINARY_DIR}/hyp)
+
+ExternalProject_Add(
```

Comment:
This is relevant only to specific board or arch. We should not build it for all boards.